modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/OAuth2RestClient.java

```diff
@@ -486,6 +487,26 @@ public List<ScopeGetModel> getAPIResourceScopes(String apiIdentifier) throws IOE
         }
     }
 
+    /**
+     * Creates a business API
+     *
+     * @param businessAPICreationModel
+     * @return ID of the created business API resource
+     */
+    public String createBusinessAPIResource(BusinessAPICreationModel businessAPICreationModel) {
```

Comment:
Rename to createDomainAPIResouce

Reply:
And also to DomainAPICreationModel

Reply:
Updated with the commit d98eafd46ac6744c57cd17961f115764a591ec00